com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/graph/ResourceGraph.java

```diff
@@ -127,6 +127,13 @@ public void add(IResource rootResource) throws CompileExceptionError {
         ResourceWalker.walk(project, rootResource, this);
     }
 
+    private void addNodeToParent(ResourceNode parentNode, ResourceNode childNode) {
+        if (parentNode.checkType(ResourceNode.Type.ExcludedCollectionProxy)) {
+            childNode.setType(ResourceNode.Type.ExcludedCollection);
```

Comment:
I made this function to have a single place where we have access to both parent and child. It gives me the ability to detect an excluded collection, and set this type in the resource graph, so we know that we have to collect dependencies for that collection in the manifest builder.